src/app/components/inline-form/index.html

```diff
@@ -151,6 +151,30 @@
 		<header><div class="sky-subsection-heading sky-margin-stacked-default">Edit form</div></header>
 			<p>When users click the Edit button, the view content fades out and the form slides in to replace the view content.</p>
 
+  <header><div class="sky-section-heading">Content</div></header>
+    <header><div class="sky-subsection-heading sky-margin-stacked-default">Primary buttons</div></header>
+  <p>The primary action on the inline form should be Save or Done.</p>
+    <stache-row>
+      <stache-column screenSmall="6">
+        <p>Use Save if the data is saved to the database immediately.</p>
```

Comment:
Not a big deal, but I'd tweak this to use active voice: "Use Save if the inline form saves data to the database immediately."

Reply:
updated